### ISSUE TITLE
Add DA adjustment and weighting with equations

### ISSUE THREAD

**Issue by kjacobsen16** [USGS-WiM/StreamStats#1060]
This will likely require an extension added to the NSSServices, similarly to the FDCTM.

After delineating a basin, I think we'll want the users to select a nearby gage - we can get a list to choose from by using the GageStatsServices [Nearest Stations on Network endpoint](https://test.streamstats.usgs.gov/docs/gagestatsservices/#/Stations/GET/NearestStationsonNetwork) to use for weighting.

@USGSPMM3 I'll see if I have more notes on the call we had a couple months ago, if you have any helpful information please send it along. I believe we were looking into the "DA_Exponent" for each equation, but I can't remember the plan there.

**Comment by USGSPMM3** [comment]
If doing just a drainage area adjustment of the gaged data (i.e. no regression equations) we can use Qu=Qg*(Au/Ag)^exp where exp is the exponent in the database. If we don't have an exponent in the database we can set exp to 1, but notify users. 

If we have a regression and a nearby gage, we should use the equation in the following image until Ken Eng publishes his new method.  In the following image Q100u is essentially the final weighted flowstat, Q100Ureg is from the regression equations, and Q100G is from the gage. Obviously these Qs would change according to the flow statistics of interest. 
![image](https://user-images.githubusercontent.com/16928289/96799001-5b842400-13bf-11eb-9191-7bb556e94a21.png)

**Comment by kjacobsen16** [comment]
Thanks @USGSPMM3 I'm a little confused about what all the variables stand for. Are you saying that we weight the basin characteristics? So for the first one Qg is the gage's charasteristic, Au is ungaged basin's drainage area, Ag is ungaged basin's drainage area and Qu is the weighted basin characteristic? Then I'm guessing for the second part, Ag and Au would mean the same.

The DA_exponent is on the Equations table, so for each regression equation there is a da_exponent. If we're using it for basin characteristics we may need to add a da_exponent somewhere else in the DB.

**Comment by USGSPMM3** [comment]
@kjacobsen16 
No, we weight the flow statistics using drainage area and the adjustment factor. In some cases we would adjust flow statistics using the da_exponent. So in the example image, Q is always discharge or a streamflow value. Qr would be regression equation, Qg would be gage statistics, and Qu would be the unregulated user-click point. In this example the 100 is for the 100-year value. Let me try to lay out a full document showing workflow and get that to you.

**Comment by USGSPMM3** [comment]
@kjacobsen16 
Let's put a temporary hold on this, Julie Kiang thought that Ken's work might be a full replacement rather than a partial replacement of these weighting methods. I've sent an email to ken

**Comment by USGSPMM3** [comment]
@kjacobsen16 
According to Ken Eng: "..keeping the old method would confuse tings, so I would suggest getting rid of it". Thus, we will only add Ken's weighting method for weighting equations with gage statistics. We will still need to add a method to weight gage stats based on drainage area in the case that we don't have regression equations. Those would be computed using Qu=Qg* (DAu/DAg)^x where x would be the DA ratio from our database or 1 if we don't have a value in the database
